pinecone/client_test.go

```diff
@@ -3,18 +3,18 @@ package pinecone
 import (
 	"context"
 	"fmt"
+	"github.com/google/uuid"
+	"github.com/pinecone-io/go-pinecone/internal/mocks"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
 	"io"
+	"log"
 	"net/http"
 	"os"
 	"reflect"
 	"strings"
 	"testing"
-
-	"github.com/google/uuid"
-	"github.com/pinecone-io/go-pinecone/internal/mocks"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-	"github.com/stretchr/testify/suite"
```

Comment:
nit: I think `goimports` should split these apart automatically right?

Reply:
I thinkkk so. This was also pre-our formatting convo, so I'll push up some changes wrt that soon!